packages/darklang/cli/cli.dark

```diff
@@ -164,6 +165,20 @@ module Darklang =
         result
 
 
+    let runFunction (fnName: String) (args: List<String>) : Int =
+      let fn = fnName |> String.dropFirst_v0 1 |> String.split "."
+      let name = fnName |> String.dropFirst_v0 1
```

Comment:
let's move this logic to _just_ after we do the "starts with @" check, so the context is visible there